config-server/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+  application:
+    name: config-server
+  cloud:
+    config:
```

Comment:
I think this section can be deleted. iit gets it from the env vars

Reply:
I think all the parts with the 'zone' shouldbeplace holders and will get the values from the env vars.
for the discovery  server  itself it can be local host